fuse/dl/losses/segmentation/loss_focalLoss.py

```diff
@@ -92,18 +92,18 @@ def __init__(self,
         :param resize_mode:             Resize mode- either using a max pooling kernel(default), or using PyTorch interpolation ('interpolate'/'maxpool')
         """
         super().__init__(pred_name, target_name, weight)
-        self.class_weights = class_weights
+        self.class_weights = class_weights # Should be deleted? -Sagi's TEMP note
```

Comment:
@mosheraboh 

Not related to this PR, but is this line should be deleted? `class_weights` are not being passed in the `__init__` method.. But maybe I'm missing something

Reply:
Yes, good catch. please delete it.